### PR TITLE
Srfinfo2vm path changes

### PR DESCRIPTION
**Pull request by jasonmotha**

### CODE REVIEW

File: SrfGen/srfinfo2vm.py [hunk 1]
```diff
@@ -792,7 +795,7 @@ def create_vm(args, srf_meta, logger_name: str = "srfinfo2vm"):
     )
 
     # proportion in ocean
-    if args.no_optimise or (xlen0 > 0 and ylen0 > 0):
+    if args.no_optimise or (xlen0 <= 0 and ylen0 <= 0):
```
Comment (jpa198) [inline SrfGen/srfinfo2vm.py]: `x<=0 or y<=0`, otherwise if only one is negative it won't optimise? (De Morgans law?)

File: SrfGen/srfinfo2vm.py [hunk 2]
```diff
@@ -1136,7 +1139,7 @@ def load_args(logger: Logger = qclogging.get_basic_logger()):
         ),
     )
     parser.add_argument(
-        "-o", "--out-dir", help="directory to place outputs", default="autovm"
+        "-o", "--out-dir", help="directory to place outputs", default="VMs"
```
Comment (jpa198) [inline SrfGen/srfinfo2vm.py]: Should this be
`Data/VMs`?

File: SrfGen/srfinfo2vm.py [hunk 3]
```diff
@@ -622,35 +624,36 @@ def gen_vm(
...
+    gen_coords(vm_dir=out_vm_dir)
     # validate
     logger.debug("Validating vm")
-    success, message = validate_vm(vm_dir)
+    success, message = validate_vm(out_vm_dir)
     if success:
-        logger.debug("VM check passed: {}".format(vm_dir))
-        sys.stderr.write("VM check OK: %s\n" % (vm_dir))
+        logger.debug("VM check passed: {}".format(vm_working_dir))
+        sys.stderr.write("VM check OK: %s\n" % (vm_working_dir))
```
Comment (jpa198) [inline SrfGen/srfinfo2vm.py]: Should have the same message?

File: SrfGen/srfinfo2vm.py (hunk 2 above)
Comment (jasonmotha) [reply SrfGen/srfinfo2vm.py]: Srfinfo2vm generally has been run from the data folder. Where does SRFgen / realisation generation run from?

File: SrfGen/srfinfo2vm.py (hunk 1 above)
Comment (jasonmotha) [reply SrfGen/srfinfo2vm.py]: It was meant to be a specific case handling a few lines up. But actually if ever either is 0 then the area is 0 then we can't really optimise it.

Comment (jpa198) [reply SrfGen/srfinfo2vm.py]: Also needs to be `not (x<=0 or y<=0)` for the logic to work out. Slowly remembering De Morgans Law (negate outside, flip conjunctive, negate particles)

File: SrfGen/srfinfo2vm.py (hunk 2 above)
Comment (jpa198) [reply SrfGen/srfinfo2vm.py]: The root directory. That way all paths are relative to the Cybershake Root

File: SrfGen/srfinfo2vm.py (hunk 1 above)
Comment (jasonmotha) [reply SrfGen/srfinfo2vm.py]: The logic now is slightly different but the cases that it doesn't optimise for are now more correct.

File: SrfGen/srfinfo2vm.py (hunk 2 above)
Comment (jasonmotha) [reply SrfGen/srfinfo2vm.py]: I'm apprehensive to change the behaviour of a script. When we refactor this one we can change it's behaviour?

**Pull request merged**